api/v1/qdrantcluster_types.go

```diff
@@ -82,6 +82,15 @@ const (
 	GPUTypeAmd    GPUType = "amd"
 )
 
+type RebalanceStrategy string
+
+//goland:noinspection GoUnusedConst
```

Comment:
would not introduce goland specifics in our files

Reply:
Otherwise goland would always recommend to remove this.

Reply:
I would prefer you fix this inside your tool, imo this is an exported enum (public)

Reply:
No. This is a useful inspection in a commonly used IDE. It's a comment that hurts no one, but prevents accidental errors by someone thinking that his can be deleted.

Reply:
I know it don't hurt, but in publicly available repos I would restrict hits for personal toolings we are using.
Next will be some VSCode annots the next will be VIM hints and other editors... where do we stop?

Maybe we should add something to our guidelines about this?